suite-common/wallet-constants/src/jws.ts

```diff
@@ -1,12 +1,5 @@
-const publicKey = {
+export const publicKey = {
     dev: '-----BEGIN PUBLIC KEY-----MFYwEAYHKoZIzj0CAQYFK4EEAAoDQgAEbSUHJlr17+NywPS/w+xMkp3dSD8eWXSuAfFKwonZPe5fL63kISipJC+eJP7Mad0WxgyJoiMsZCV6BZPK2jIFdg==-----END PUBLIC KEY-----',
     codesign:
         '-----BEGIN PUBLIC KEY-----MFYwEAYHKoZIzj0CAQYFK4EEAAoDQgAES7MbBzU/v5BsljkTM8Mz0Jsk+Nn5n2wH\no2/+MUI3TgCVdTbEHhn3HXaY7GJ6TLyWqxn+pIDY9wUUAyUqOStTUQ==-----END PUBLIC KEY-----',
 };
-
-export const getJWSPublicKey = () => {
-    const isCodesignBuild = process.env.IS_CODESIGN_BUILD === 'true';
-    console.log(`Bundling ${isCodesignBuild ? 'production' : 'develop'} public key.`);
```

Comment:
This is removed without replacement, right? It's probably fine, not sure if anyone checking this log. If so, it would be ease to put it back on the place where it was used (but keep it outside this method). cc @komret

Reply:
Yes, log was removed, I forgot to mention that in description. Hopefully it's fine, if now, I'll return it in a follow-up.
https://github.com/trezor/trezor-suite/issues/11817